src/vm/runners/cairo_runner.rs

```diff
@@ -410,6 +411,35 @@ impl CairoRunner {
         Ok(())
     }
 
+    pub fn get_perm_range_check_limits(
```

Comment:
why this method is in Runner? Looks like it only needs vm

Reply:
It was originally within `CairoRunner`.